juntagrico/templatetags/depot_extras.py

```diff
@@ -44,7 +45,9 @@ def get_types_by_size(subscriptions, size):
         parts = subscriptions.active_parts
     else:
         # case 2: queryset of subscriptions is passed
-        parts = SubscriptionPart.objects.filter(subscription__in=subscriptions, activation_date__isnull=False, deactivation_date__isnull=True)
+        now = timezone.now().date()
+        parts = SubscriptionPart.objects.filter(subscription__in=subscriptions, activation_date__lte=now).\
+            filter(Q(deactivation_date__isnull=True) | Q(deactivation_date__gte=now))
```

Comment:
do we not have a q object method for that?
i will merge it for now but please do a fix for that